src/vfs/public.c

```diff
@@ -160,10 +199,28 @@ bool vfs_mkparents(const char *path) {
 }
 
 char* vfs_repr(const char *path, bool try_syspath) {
-	char buf[strlen(path)+1];
-	path = vfs_path_normalize(path, buf);
-	VFSNode *node = vfs_locate(vfs_root, path);
+    char buf[strlen(path)+1];
+/*
```

Comment:
Again, large comment blocks of dead code should be deleted outright. No point keeping them around.